test/TestServerProjects/custom-baseUrl-paging/Generated/Operations/PagingOperations.cs

```diff
@@ -365,6 +365,84 @@ public Response<ProductResult> GetPagesPartialUrlNextPage(string nextLink, Cance
                 throw;
             }
         }
+        internal HttpMessage CreateGetPagesPartialUrlOperationNextNextPageRequest(string nextLink)
```

Comment:
Why do we now generate this?

Reply:
This is due to: https://github.com/Azure/autorest.modelerfour/issues/202
When that issue is fixed, this code should be removed, and no paging code should need to be changed.